src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -66,15 +68,49 @@ export default class DashboardTable extends React.Component {
         this.setState({
             currentPage: newPage,
         }, () => {
-            this.props.loadTableData(this.state.currentPage, this.props.isCertified);
+            this.props.loadTableData(this.state.currentPage, this.props.isCertified, this.getCategory(), this.state.sortDirection);
         });
     }
 
     reload(){
-        this.props.loadTableData(this.state.currentPage, this.props.isCertified);
+        this.props.loadTableData(this.state.currentPage, this.props.isCertified, this.getCategory(), this.state.sortDirection);
         buildRow();
     }
 
+    sortTable(direction, column) {
+        // the table sorting changed
+        this.setState({
+            sortDirection: direction,
+            sortColumn: column
+        }, () => {
+            // re-display the data
+            this.props.loadTableData(this.state.currentPage, this.props.isCertified, this.getCategory(), this.state.sortDirection);
+            this.buildRow();
+        });
+    }
+
+    getCategory(){
+        switch(this.state.sortColumn){
+            case 1:
+                return 'agency';
+                break;
+            case 2:
+                return 'reporting';
+                break;
+            case 3:
+                return 'submitted_by';
+                break;
+            case 4: 
```

Comment:
I know it breaks the order but would it make more sense to just remove this case altogether since it does the exact same thing as `default`?